src/rest_api/classes/generic_fields.clj

```diff
@@ -12,22 +13,22 @@
   (let [id-kw (first (filter #(= (name %) "id") (keys object)))
         role (namespace id-kw)]
     {:data (when-let [ghs ((keyword role "gene") object)]
-             (for [gh ghs :let  [gene ((keyword (str role ".gene") "gene") gh)]]
+             (for [gh ghs :let [gene ((keyword (str role ".gene") "gene") gh)]]
```

Comment:
Also, in the `for`, `(keyword (str role ".gene")` could be pulled up to a var in the outer let to avoid recomputing it each time.